src/server/controllers/cards.ts

```diff
@@ -41,9 +41,11 @@ export const deleteCard = (req: Request, res: Response) => {
 }
 
 export const fetchFlashcards = (_req: Request, res: Response) => {
+  const minCards = 3
+  const maxCards = 10
```

Comment:
Diese könntest du in Java-Manier auf der höchsten Ebene im File deklarieren und MIN_CARDS und MAX_CARDS verwenden. Das würde im Code noch etwas aussagekräftiger aussehen.